src/common/models/Projection.ts

```diff
@@ -181,6 +188,11 @@ export class Projection {
       this.cumulativeNegativeTests,
     );
 
+    const cumulativeActualDeaths = actualTimeseries.map(
+      row => row?.cumulativeDeaths || null,
+    );
```

Comment:
Maybe we should smooth the `cumulativeActualDeaths` with `smoothCumulatives`? If the series has a hole, maybe we should use the value of the previous day (we know that the series should monotonically increase day to day)

**Example**

```
Date        CumulativeDeaths
----------------------------
2020-06-01  10
2020-06-02  12
2020-06-03  null // we know it must be at least 12
2020-06-04  16
// ...
```

Reply:
Oh, yes.  Using `smoothCumulatives()` is a good idea.  It would fill that gap with 14 (split the difference between the day before and the day after).